src/content/developers/docs/apis/backend/index.md

```diff
@@ -45,6 +45,11 @@ These libraries abstract away much of the complexity of interacting directly wit
 
 - [cloudflare-eth.com](https://cloudflare-eth.com)
 
+**DataHub by Figment -** **_Web3 API services with Ethereum mainnet and testnets._**
```

Comment:
```suggestion
**DataHub by Figment -** **_Web3 API services with Ethereum Mainnet and testnets._**
```

Reply:
Small change here as per [EIP-2228](https://eips.ethereum.org/EIPS/eip-2228) - (_The Ethereum network with network ID 1 and chain ID 1 is named Ethereum Mainnet._)